File: JobsAPI/Models/Job.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace JobsAPI.Models
{
    public class Job
    {
        public int Id { get; set; }
        public string description { get; set; }
        public string redirect_url { get; set; }
        public decimal salary_max { get; set; }
        public string location { get; set; }
        public string title { get; set; }
        public decimal salary_min { get; set; }
        public string company { get; set; }
        public string created { get; set; }
        public State? state { get; set; }
        public int stateid { get; set; }
        public City? city { get; set; }
        public int cityid { get; set; }
        public Category? category { get; set; }
        public int categoryid { get; set; }
        public user? user { get; set; }
        public int? userid { get; set; }
    }
}

File: JobsFront/src/app/_interfaces/Job.ts
import { Category } from "./Category";
import { City } from "./City";
import { State } from "./State";

  export interface Job {
    id: number;
    description: string;
    redirect_url: string;
    salary_max: number;
    location: string;
    title: string;
    salary_min: number;
    company: string;
    created: string;
    state: State;
    stateid: number;
    city: City;
    cityid: number;
    category: Category;
    categoryid: number;
    userid: number;
    
}
  